ingest-app/src/cmr/ingest/api/generic_documents.clj

```diff
@@ -134,6 +127,34 @@
                version
                (.getMessage e))))))
 
+(defn-timed save-document
+  "Store a concept in mdb and indexer. Return concept-id, and revision-id."
+  [context concept]
+  (let [{:keys [concept-id revision-id]} (mdb/save-concept context concept)
+        doc-name (get-in concept [:extra-fields :document-name])]
+    {:name doc-name
+     :concept-id concept-id
+     :revision-id revision-id}))
+
+(defn ingest-document
+  "Ingest the concept into the database and the indexer through the database."
+  [context concept headers]
+  (info (format "Ingesting collection %s from client %s"
+                (api-core/concept->loggable-string concept)
```

Comment:
this is a lot of content, should we not just log the ID or something short?

Reply:
I copied what we do for collections and services.